dockstore-client/pom.xml

```diff
@@ -90,6 +90,10 @@
                     <groupId>org.mockito</groupId>
                     <artifactId>mockito-core</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Common enough library that is should (if it isn't already) be managed in the root pom